src/Evolution/Executables/GrMhd/ValenciaDivClean/CMakeLists.txt

```diff
@@ -20,34 +21,97 @@ set(LIBS_TO_LINK
   )
 
 add_spectre_parallel_executable(
+  EvolveFishboneMoncriefDisk
   EvolveValenciaDivClean
+  Evolution/Executables/GrMhd/ValenciaDivClean
+  EvolutionMetavars<RelativisticEuler::Solutions::FishboneMoncriefDisk>
+  "${LIBS_TO_LINK}"
+  )
+
+add_spectre_parallel_executable(
+  EvolveTovStar
   EvolveValenciaDivClean
   Evolution/Executables/GrMhd/ValenciaDivClean
-  EvolutionMetavars
+  EvolutionMetavars<RelativisticEuler::Solutions::TovStar<gr::Solutions::TovSolution>>
   "${LIBS_TO_LINK}"
   )
 
-set(LIBS_TO_LINK
-  CoordinateMaps
-  DiscontinuousGalerkin
-  DomainCreators
-  GeneralRelativitySolutions
-  GrMhdAnalyticData
-  Hydro
-  IO
-  Informer
-  Limiters
-  LinearOperators
-  MathFunctions
-  Time
-  Utilities
-  ValenciaDivClean
+add_spectre_parallel_executable(
+  EvolveAlfvenWave
+  EvolveValenciaDivClean
+  Evolution/Executables/GrMhd/ValenciaDivClean
+  EvolutionMetavars<grmhd::Solutions::AlfvenWave>
+  "${LIBS_TO_LINK}"
+  )
+
+add_spectre_parallel_executable(
+  EvolveBondiMichel
+  EvolveValenciaDivClean
+  Evolution/Executables/GrMhd/ValenciaDivClean
+  EvolutionMetavars<grmhd::Solutions::BondiMichel>
+  "${LIBS_TO_LINK}"
+  )
+
+add_spectre_parallel_executable(
+  EvolveKomissarovShock
```

Comment:
(Picking one at random.)  These executables will not be built by any of the general make targets (like "all" or "test-executables"), correct?  I'm just afraid of unintentionally trying to build several of these in parallel and locking up my machine as the gccs try to use 100 GB of RAM.

Reply:
That is correct, these are not built unless you specify the exact target (except for one of each flavor, that have input files in `tests/InputFiles`

Reply:
should we have Travis build them occasionally?

Reply:
How do you plan on having Travis do that? (I'm not saying it's impossible, I'm just wondering what your implementation strategy is)

Reply:
Perhaps build them `weekly` (https://docs.travis-ci.com/user/cron-jobs/) on top of the daily develop build.
We could also set up regression tests for all of them that run daily or weekly and compare their output to some h5 data file.